tests/Platforms/AbstractPlatformTestCase.php

```diff
@@ -592,6 +592,18 @@ public function testGetDefaultValueDeclarationSQLForDateType(): void
         }
     }
 
+    public function testGetDefaultValueDeclarationSQLForJsonType(): void
```

Comment:
Please do not add tests that make assertions against SQL. They are error-prone and they do not prove that the functionality you're trying to implement actually works. Instead, please implement an integration (a.k.a. functional) test that reproduces your scenario. See more details in the [documentation](https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/testing.html#integration-tests).